_data/sidebars/es_lb4_sidebar.yml

```diff
@@ -13,9 +13,9 @@ translated: true
 children:
 
 - title: 'Iniciando'
-  url: Getting-started.html
+  url: iniciando.html
```

Comment:
I think the url should stay as the English one. From [Translating articles](https://loopback.io/doc/en/contrib/Translating_articles.html#procedure):

> Change permalink from `/doc/en/lb2/Foo.html` to `/doc/xx/lb2/Foo.html`.



>